proposals/field-keyword.md

```diff
@@ -662,4 +662,50 @@ class C
 
 For reference, fields for *primary constructor parameters* are generated in similar cases - see [sharplab.io](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA+ABADAAgwRgDoARASwEMBzAOwgGcAXUsOgbgFgAoLjAJhwDCACgjAAVjDAMcAM1IwANgBMAlFwDeXHNrwocAWSFrOOnJpOmdxGMACulQgEE6dGFAZC5ipTlJ0c1LYKCiocFtoAvlxR3JyMULZSOADKIuKS0l7KxuamGHqGxqa5ltrWdg7Oru6e8sq+/oHBoVo6MRFAA).
 
-**Recommendation**: The backing field is generated when `field` is used only in omitted calls to *conditional methods*.
\ No newline at end of file
+**Recommendation**: The backing field is generated when `field` is used only in omitted calls to *conditional methods*.
+
+### Require overriding both accessors
+
+An auto-implemented property that overrides a virtual read-write property is currently required to override both accessors. Should that requirement be extended to accessors that use `field`?
+
+```csharp
+class A
+{
+    public virtual object P1 { get; set; }
+    public virtual object P2 { get; set; }
+    public virtual object P3 { get; set; }
+}
+
+class B : A
+{
+    public override object P1 { get; }          // error: must override both accessors
+    public override object P2 { get => null; }  // ok
+    public override object P3 { get => field; } // ok?
+}
+```
+
+### Default interface implementation
+
+Is `field` supported in a default interface implementation of a property, for `static` properties at least?
```

Comment:
yes... i'm not sure why this would be an open question.  This is already legal: https://sharplab.io/#v2:C4LgTgrgdgPgllYBTMAzAhgYyQAgJICwAUAN7E4U4ACAjAGw4LA4AKOJOA5ksANw4BnHvwC+xMUWLEqAJhwARYmSKVq9agBYcAWXQIAFAEp2EkUA

Why would we not support `field` here given that there *is* a backing field the compiler generates.

It certainly is not legal in a normal interface as they don't allow for state though (even with DIMs).